reliability/README.md

```diff
@@ -128,13 +132,52 @@ Set environment virable SLACK_API_TOKEN before running the test. Contact qili@re
 
 If you're in [CoreOS](coreos.slack.com) workspace, but you want to use your own slack channel, create a slack channel and install App `OCP Reliability` which already exists in CoreOS workspace.
 
 If you want to use your own App(slack_api_token), create an [app](https://api.slack.com/apps?new_granular_bot_app=1) and add a bot to it on slack. Slack Bot Token Scopes permissions are [channels:read] [chat:write] [groups:read] [im:read] [mpim:read]. You will get a token after the app is installed to a workspace. Install the app to your channel. Set the token to SLACK_API_TOKEN environment variable.
 
+### Kraken Integration
+Configure Kraken scenario(s) to trigger error injection during the Reliability test.
+
+The below configuration example enables the Kraken integration by setting `kraken_enable: True`. 
+`kraken_scenarios` defines the [Kraken scenario(s)](https://github.com/cloud-bulldozer/kraken-hub/blob/main/README.md#supported-chaos-scenarios) you want to run during Reliability test. Refer to each Kraken scenario's document for the supported `parameters`, e.g [pod-scenarios](https://github.com/cloud-bulldozer/kraken-hub/blob/main/docs/pod-scenarios.md#supported-parameters).
+
+`interval_unit` and `interval_number` schedules how often the Kraken scenario is triggered. `start_date` and `end_date` with `timezone` are used to limit the time range to schedule the Kraken scenario. The scheduler feature makes use of [apscheduler](https://apscheduler.readthedocs.io/en/latest/modules/triggers/interval.html#module-apscheduler.triggers.interval).
+
+Though Kraken senarios support to run the error injection multiple times with iterations or in daemon mode as parameter, in Reliability test, you may want to get notification of start and end time of each error injection, to check what errors happen in Reliability test during the error injection period. In this case, configure in Reliability test to trigger the Kraken snario with `interval_unit` and `interval_number` is recommended.
+
+If [Slack Integration](#Slack-Integration) is enabled, notification of the Kraken scenarios' start and end time as well as result will be send to the slack channel. You can check what errors of Reliability test happen during the Kraken error injection.
+
+```yaml
+    kraken_scenarios: 
+      - name: pod-scenarios_etcd
+        scenario: "pod-scenarios"
+        interval_unit: minutes # weeks,days,hours,minutes,seconds
+        interval_number: 8
+        # start_date: "2021-10-28 17:36:00" # Optional. format: 2021-10-20 10:00:00.
+        # end_date: "2021-10-28 17:50:00" # Optional.
+        # timezone: "Asia/Shanghai" # Optional. e.g. US/Eastern. Default is "UTC
+      - name: pod-scenarios_monitoring
+        scenario: "pod-scenarios"
+        interval_unit: minutes # weeks,days,hours,minutes,seconds
+        interval_number: 10
+        parameters:
```

Comment:
Maybe also allow setting the replicas and number of runs  - https://github.com/cloud-bulldozer/kraken-hub/blob/main/docs/pod-scenarios.md#supported-parameters? Thoughts?

Reply:
@chaitanyaenr Ravi, thank you for your time to review. To be clear, user can add any parameter to under `parameters` in the configuration file, that's dynamic. The listed ones in the document are just for an example. I will update the document to explain this clearer.